vllm/attention/backends/torch_sdpa.py

```diff
@@ -86,10 +86,13 @@ class TorchSDPAMetadata(AttentionMetadata, PagedAttentionMetadata):
     # For chunked prefill only
     max_query_len: Optional[int] = None
     max_kv_len: Optional[int] = None
-    query_start_loc: Optional[torch.Tensor] = None
```

Comment:
why does this file require changes?

Reply:
It is a naming conflict. The V1 model runner use ```query_start_loc``` for logits indexing specially, contains all tokens in a batch. But in ```torch_sdpa```, ```query_start_loc``` contains prefill tokens only, so rename it to ```prefill_query_start_loc```.

Reply:
I would try to avoid the renaming in torch sdpa instead of the global change